src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrPatchHandler.cs

```diff
@@ -123,9 +123,11 @@ public class VmrPatchHandler : IVmrPatchHandler
             };
         }
 
-        var filters = new List<string>();
-        filters.AddRange(mapping.Include.Select(p => $":(glob,attr:!{VmrInfo.IgnoreAttribute}){p}"));
-        filters.AddRange(mapping.Exclude.Select(p => $":(exclude,glob,attr:!{VmrInfo.KeepAttribute}){p}"));
+        List<string> filters =
+        [
+            .. mapping.Include.Select(GetInclusionRule),
```

Comment:
I don't love this new `spread element` syntax but I guess I'd get used to it if you think it's the way to go.
Not sure if this is more readable than 
```
mappings.Include.Select(GetInclusionRule)
  .Concat(mapping.Exclude.Select(GetExclusionRule))
  .ToList()
```
it's definitely shorter tho

Reply:
I've been using this a lot in TypeScript and I always thought C# misses it so my opinion might be influenced. But it's very hard to live without it once you internalize using it.

To me, the blob of LINQ is harder to parse as I need to read into what we start with and what we end up with. But the `[ ]` block somehow immediately means an array.